docs/PayPal/integration.md

```diff
@@ -127,25 +128,19 @@ The `id` field of the response contains the order ID to pass to your client.
 
 ### 6. Create a request object for launching the PayPal flow
 
-Configure your `PayPalRequest` and include the order ID generated [step 5](#5-create-an-order):
+Configure your `PayPalWebCheckoutRequest` and include the order ID generated [step 5](#5-create-an-order):
 
 ```kotlin
-val payPalRequest = PayPalRequest("<ORDER_ID>")
+val payPalWebCheckoutRequest = PayPalWebCheckoutRequest("<ORDER_ID>")
 ```
 
 ### 7. Approve the order through the Payments SDK
 
-When a user taps the PayPal button created above, approve the order using your `PayPalClient`.
+When a user initiates the PayPal payment flow through your UI, approve the order using your `PayPalWebCheckoutClient`
 
-Attach your PayPal button to the PayPal payment flow:
-
-```kotlin
-findViewById<View>(R.id.payPalButton).setOnClickListener {
-    payPalClient.checkout(payPalRequest) 
-}
-```
+Call `payPalClient.approveOrder(payPalWebCheckoutRequest)` to approve the order.
```

Comment:
We still don't have parity between iOS and Android on this it looks like iOS is `start` and Android is `approveOrder` - do we still have a ticket open to get those aligned?

Reply:
@sarahkoop i remember having the discussion but dont actually remember what we decided. Seems to me that `start` makes a lot of sense, since we are in fact, starting the checkout. I'll change it